docs/source/embedded-models.rst

```diff
@@ -0,0 +1,55 @@
+Embedded models
+===============
+
+Use :class:`~django_mongdob.fields.EmbeddedModelField` to structure your data
+using `embedded documents
+<https://www.mongodb.com/docs/manual/data-modeling/#embedded-data>`_.
+
+The basics
+----------
+
+Let's consider this example::
+
+   from django_mongodb_backend.fields import EmbeddedModelField
+
+   class Customer(models.Model):
+       name = models.CharField(...)
+       address = EmbeddedModelField("Address")
+       ...
+
+   class Address(models.Model):
+       ...
+       city = models.CharField(...)
+
+
+The API is similar to that of Django's relational fields::
+
+   >>> Customer.objects.create(name="Bob", address=Address(city="New York", ...), ...)
+   >>> bob = Customer.objects.get(...)
+   >>> bob.address
+   <Address: Address object>
+   >>> bob.address.city
+   'New York'
+
+Represented in BSON, Bob's structure looks like this:
+
+.. code-block:: js
+
+   {
+     "_id": ObjectId(...),
+     "name": "Bob",
+     "address": {
+       ...
+       "city": "New York"
+     },
+     ...
+   }
+
+Querying ``EmbeddedModelField``
+-------------------------------
+
+You can query into an embedded model using the same double underscore syntax
+as relational fields. For example, to retrieve all customers who have an
+address with the city "New York"::
```

Comment:
We should also emphasize that you can query using a dictionary, but we that's not 

```python
Customer.objects.filter(address={..."city": "New York"})
```

Reply:
I don't think it works. e.g. `Book.objects.filter(author={"name": "Shakespeare"})` gives `db.model_fields__book.aggregate([{'$match': {'$expr': {'$eq': ['$author', {'name': 'Shakespeare'}]}}}])`. I think you're thinking of Djongo's query syntax.

Reply:
@Jibola You can use a lookup dictionary, however, you have to specify the exact dictionary of the embedded document. In my example, author has other fields besides `name` so it didn't work. It makes sense now that I think about it, but it wasn't the intuitive result I expected. Would you like me to test and document this pattern? There's probably no use case for querying this way if we start including `_id` in embedded models (see Slack discussion).